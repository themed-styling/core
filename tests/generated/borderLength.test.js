import 'jest-styled-components'

import { borderLength } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(borderLength, 'borderLength')
