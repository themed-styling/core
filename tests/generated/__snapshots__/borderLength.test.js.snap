// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`borderLength renders 0 1`] = `
Array [
  .c0 {
  border-length: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 0;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders function 1`] = `
Array [
  .c0 {
  border-length: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: has;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: has;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders invalid theme path 1`] = `
Array [
  .c0 {
  border-length: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders negative number 1`] = `
Array [
  .c0 {
  border-length: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    border-length: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-length: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    border-length: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-length: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-length: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-length: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    border-length: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    border-length: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    border-length: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    border-length: July;
  }
}

@media April {
  .c0 {
    border-length: August;
  }
}

@media January {
  .c0 {
    border-length: September;
  }
}

@media April {
  .c0 {
    border-length: November;
  }
}

@media June {
  .c0 {
    border-length: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    border-length: July;
  }
}

@media April {
  .c0 {
    border-length: August;
  }
}

@media January {
  .c0 {
    border-length: September;
  }
}

@media April {
  .c0 {
    border-length: November;
  }
}

@media June {
  .c0 {
    border-length: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-length: January;
  }
}

@media April {
  .c0 {
    border-length: December;
  }
}

@media January {
  .c0 {
    border-length: April;
  }
}

@media April {
  .c0 {
    border-length: May;
  }
}

@media June {
  .c0 {
    border-length: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    border-length: January;
  }
}

@media April {
  .c0 {
    border-length: December;
  }
}

@media January {
  .c0 {
    border-length: April;
  }
}

@media April {
  .c0 {
    border-length: May;
  }
}

@media June {
  .c0 {
    border-length: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders postitive number 1`] = `
Array [
  .c0 {
  border-length: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders string 1`] = `
Array [
  .c0 {
  border-length: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: July;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: June;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: June;
}

<div
    className="c0"
  />,
]
`;

exports[`borderLength renders valid theme path 1`] = `
Array [
  .c0 {
  border-length: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  border-length: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
